lib/src/views/misc/swipeable_stack.dart

```diff
@@ -89,7 +89,7 @@ class SwipeableStack extends StatelessWidget {
                 child: ClipRRect(
                   borderRadius: BorderRadius.circular(20.0),
                   child: GestureDetector(
-                    onTap: () => print('clicked the photo'), // ignore: avoid_print
+                    onTap: () => print('NOT IMPLEMENTED'), // ignore: avoid_print
```

Comment:
Perhaps add a TODO here so these show up in IntelliJ's TODO pane.

Reply:
Same goes for all similar occurrences in `profile_view`.

Reply:
Added all the missing TODO comments in e9f6d88973c84d8f95e2940e2b17338a59e30a4c.